src/helloworld.wast

```diff
@@ -0,0 +1,32 @@
+;; Copied from the wasmtime WASI tutorial
+;; https://github.com/bytecodealliance/wasmtime/blob/1f8921ef09667dc4319c1b65ebb1ce993e19badf/docs/WASI-tutorial.md#web-assembly-text-example
+;;
+;; wat2wasm helloworld.wast
+
+(module
```

Comment:
could we wat2wasm when building the kernel? it seems super easy to accidentally end up with a binary wasm dep in the kernel that doesn't match some associated source

Reply:
That'd be pretty solid. We could also use a pure-rust wat2wasm converter (there are a few iirc) and just run it in our build script.
I just did it this way because it was the easiest, but not checking in binaries is probably a good plan.